### ISSUE TITLE
Missing reviewer details

### ISSUE THREAD

**Issue by bastistician** [retostauffer/ojstransition#4]
Thanks for the OJS upgrade.

In the new version, reviewer search seems to be limited to searching by name. I'd like to at least *see* the e-mail address of a match to be sure that I am inviting the right person. The problem is that names are not unique and e-mail addresses become outdated. In OJS2, I could verify the account details of reviewers before invitation (e.g., affiliation).

**Comment by bastistician** [comment]
I can see the e-mail address after the fact if I click on Activity Log > View Email. Luckily in my case, I've emailed the right person and address.

**Comment by retostauffer** [comment]
Hy @bastistician 

Fair point, I need to have a look at it. If I remember correct it is unfortunately not possible to _search_ by email but I will double-check it. If so I will try to display the email address with the name in the search results. Would that work for now?

We'll also have a look on the upcoming release and make sure this will work again (in a few months).

**Comment by bastistician** [comment]
Yes, being able to see the e-mail address would already be an important improvement. Interestingly, if I use the search field in "enroll existing user as reviewer", e-mail addresses are displayed, but I couldn't find the existing reviewer there to verify the address. That search also does not seem to support "firstname lastname" searches, only one word.

**Comment by retostauffer** [comment]
@bastistician : the current development version of OJS brings _some_ improvements.

However, in the current version we have not found an easy way to display the email address alongside with the name of our reviewers. A request has been sent to the developers, I hope that we can address this in the future and improve the current situation (no promise for now; https://github.com/pkp/pkp-lib/issues/7455).

Thanks for reporting, very valuable input.

**Comment by retostauffer** [comment]
Closed due to inactivity.
